_specs/ecip-1078.md

```diff
@@ -0,0 +1,101 @@
+
+After HARD_FORK_BLOCK, apply the following changes:
+
+- Disable [EIP-2200](https://eips.ethereum.org/EIPS/eip-2200)
+- Enable [EIP-1283](https://eips.ethereum.org/EIPS/eip-1283) with [EIP-1706](https://eips.ethereum.org/EIPS/eip-1706) instead.
+- Enable [https://specs.corepaper.org/50-selfbalance50-SELFBALANCE
+- Enable [ECIP-1080](https://github.com/ethereumclassic/ECIPs/issues/266)
+
+
+### Rationale
+
+[ECIP 1061: Aztlán EVM and Protocol Upgrades (Yingchun Edition)](https://ecips.ethereumclassic.org/ECIPs/ecip-1061) as defined is "broken" and it would be inadvisable to activate on the ETC mainnet in
+its current state.
+
+See [EIP 1884: Repricing for trie-size-dependent opcodes](https://eips.ethereum.org/EIPS/eip-1884) for
+the original rationale for the gas-repricings for ETH.
+
+These changes are sensible, but they [broke 680 deployed Aragon smart contracts](https://www.coindesk.com/ethereums-istanbul-upgrade-will-break-680-smart-contracts-on-aragon) along with various other deployed
+smart contacts by applying the repricings retroactively and unconditionally.
+
+The new SELFBALANCE opcode does not break backwards compatibility.  For compatibility with ETH it is both
+necessary and desirable.
+
+Short of imminent and catatrophic danger to the ETC network, making retroactive changes which break
+backwards compatibility is counter to ETC values and philosophy.
+
+Gas-repricings to balance EVM gas prices with real world resource usage is entirely rational and is
+happening on the ETC protocol too, but will be done in a backwards-compatible ways wherever possible.
+Unconditional retroactive ("emergency") repricings would only happen when the network has been halted
+or has been rendered unusable.
+
+An example of such a situation occurred following the Shanghai attacks on ETH and ETC networks:
+
+[ECIP 1015: Long-term gas cost changes for IO-heavy operations to mitigate transaction spam attacks](https://ecips.ethereumclassic.org/ECIPs/ecip-1015), which has ETC's equivalent to [ETH Tangerine Whistle](https://eips.ethereum.org/EIPS/eip-608),
+with both chains implementing [EIP 150: Gas cost changes for IO-heavy operations](https://eips.ethereum.org/EIPS/eip-150).
+
+
+### Implementation
+
+Adoption of the content of this ECIP requires a hard fork as it introduces changes that are not backward compatible.
+
+The following clients with Ethereum Classic support implement the _Istanbul_ features currently:
+
+- Parity Ethereum
+- Multi-Geth
+- Hyperledger Besu
+
+The author is unaware if any of these clients have implemented [EIP-1706](https://eips.ethereum.org/EIPS/eip-1706), but it is a very simple change, analogous to [EIP-2200](https://eips.ethereum.org/EIPS/eip-2200), which all the
+codebases have already implemented.
+
+Similarly [ECIP-1080](https://github.com/ethereumclassic/ECIPs/issues/266) has not been implemented in any of these codebases, but is just a "remix" of code which they have already implemented.
+
+
+## Copyright
```

Comment:
To comply with the Apache-2.0 license, please include the content of the NOTICE file here: https://corepaper.org/NOTICE